README.md

```diff
@@ -22,6 +22,10 @@ Labels commonly include operating system name, version, and architecture.
 | Red Hat Enterprise Linux 8 | `RedHatEnterprise` | `8.0`          | `amd64`  |
 | Scientific 6.10            | `Scientific`       | `6.10`         | `amd64`  |
 | Scientific 7.7             | `Scientific`       | `7.7`          | `amd64`  |
+| SLES 11.3                  | `SUSE LINUX`       | `11.3`          | `amd64`  |
```

Comment:
Labels that contain embedded spaces are much more difficult to manage than labels without embedded spaces.  Since the lsb_release `Distributor ID` field is defined with rules that preclude a space in the ID, I'd like this to be `SUSE` rather than `SUSE LINUX`.
```suggestion
| SLES 11.3                  | `SUSE`             | `11.3`          | `amd64`  |
```